compiler/lexer.nim

```diff
@@ -874,6 +874,10 @@ proc rawGetTok(L: var TLexer, tok: var TToken) =
     of '`':
       tok.tokType = tkAccent
       inc(L.bufpos)
+    of '_':
```

Comment:
That's not correct, you need to ensure that ``_foo`` doesn't parse as tkSymbol, tkSymbol.